recipes/cyvcf2/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set name = "cyvcf" %}
+{% set name = "cyvcf2" %}
```

Comment:
mh, should we remove the old package then?

Reply:
I was going to ask you that... I think it would be a good idea. It is just the last one for ten days ago that had this name typo.

Reply:
Ok, let me remove the last one.
Thanke Andreas!